app/controllers/calendarevents_controller.rb

```diff
@@ -0,0 +1,69 @@
+class CalendareventsController < ApplicationController
+  before_action :set_calendarevent, only: %i[show edit update destroy]
+
+  # GET /calendarevents or /calendarevents.json
+  def index
+    @calendarevents = Calendarevent.all
+  end
+
+  # GET /calendarevents/1 or /calendarevents/1.json
+  def show; end
+
+  # GET /calendarevents/new
+  def new
+    @calendarevent = Calendarevent.new
+  end
+
+  # GET /calendarevents/1/edit
+  def edit; end
```

Comment:
do the "show" and "edit" functions need to have anything or are they just left overs from inheritance?